throw.go

```diff
@@ -35,7 +35,7 @@ var commandThrow = cli.Command{
 func doThrow(c *cli.Context) error {
 	optHostID := c.String("host")
 	optService := c.String("service")
-	optMaxRetry := c.Int("max-retry")
```

Comment:
I forgot to change here when I changes option name from `--max-retry` to `--retry` on https://github.com/mackerelio/mkr/pull/159/commits/f985799ea471603b3077c66d5b8aac4246872b06